openreview/venue_request/process/invitationStatusProcess.js

```diff
@@ -0,0 +1,25 @@
+function(){
+  var or3client = lib.or3client;
+
+  var GROUP_PREFIX = '';
+  var SUPPORT_GROUP = GROUP_PREFIX + '/Support';
+
+  or3client.or3request(or3client.notesUrl + '?id=' + note.forum, {}, 'GET', token)
+  .then(function(result) {
+
+    var forumNote = result.notes[0];
+
+    var message = {
+      groups: note.readers,
+      ignoreGroups: [note.tauthor, SUPPORT_GROUP],
+      subject: 'Comment posted to your request for service: ' + forumNote.content.title,
+      message: 'A comment was posted to your service request. \n\nComment title: ' + note.content.title + '\n\nComment: ' + note.content.comment + '\n\nTo view the comment, click here: ' + baseUrl + '/forum?id=' + note.forum + '&noteId=' + note.id
```

Comment:
The note is going to be an instance of Paper_Matching_Setup_Status, Recruitment_Status and Remind_Recruitment_Status. Could you format the email based on the schema of these notes? 

The subject should say "Paper Matching Setup Status for service...." or "Recruitment Status for service..."

Reply:
Done.